PixivConfig.py

```diff
@@ -92,8 +92,8 @@ class PixivConfig():
         ConfigItem("Settings", "downloadAvatar", False),
         ConfigItem("Settings", "useSuppressTags", False),
         ConfigItem("Settings", "tagsLimit", -1),
-        ConfigItem("Settings", "writeImageInfo", False),
         ConfigItem("Settings", "writeImageJSON", False),
+        ConfigItem("Settings", "writeImageInfo", False),
```

Comment:
Moving this now separates the two JSON configs from each other. Was there a good reason for this?

Reply:
because `writeImageJSON` is the original data from the api, while `writeImageInfo` is just a few texts extracted from it.

so making the original above others is trying to let users know that the next few settings are just a subset of it.
however, the other settings need some extra info too.

Reply:
historically, the `writeImageInfo` was come up first, then only followed by `writeImageJSON` when someone ask for the raw json.

Reply:
Is the `writeImageInfo` the same as `writeImageJSON` when no `writerawjson`?